mantidimaging/core/process_list/process_list.py

```diff
@@ -120,18 +121,23 @@ def store(self, func, *args, **kwargs):
         Store a function in the process list to be executed later. The arguments must be literal values.
         Note: More arguments can be appended when the funciton is called for execution.
 
-        :param func: This works with a function reference, because metadata needs to be read from the funciton.
+        :param func: This works with a function reference, because metadata needs to be read from the function.
         :param args: Arguments that will be forwarded to the function. They must be literal values.
         :param kwargs: Keyword arguments that will be forwarded to the function. They must be literal values.
         """
         self._store_func(func, args, kwargs)
 
     def _store_func(self, func, args, kwargs):
```

Comment:
I know this is not related to these changes but would it be better to define a type that captures the function/arguments into a single object that is callable rather than having to store the stringified version of the function + signature?

Reply:
I think the advantage of the stringified version was the ability to read and edit them as plain text. That being said I'm not adversed to looking at other ways of doing this.

Reply:
Fair point - I don't know enough about the requirements to say for sure.